.travis.yml

```diff
@@ -1,2 +1,9 @@
 language: ruby
+services:
+  - rabbitmq
+before_script:
```

Comment:
This is a good change! However, I think the expensive end to end tests should happen after the less expensive unit tests, so I would use `after_script` or even `after_success`.